### PR TITLE
Prepare for E2E tests

### PR DESCRIPTION
**Pull request by tirr-c**

### CODE REVIEW

File: util/e2e.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+echo -n "*** Running Django server..."
+cd backend
+./manage.py runserver >../backend.stdout 2>../backend.stderr &
```
Comment (skystar-p) [inline util/e2e.sh]: good &

**Comment by skystar-p** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit 588cff0 has been approved by `skystar-p`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit 588cff0731701ff47cc8ad99f4e7f860f49a39c6 with merge 8bcdef4755d7297c4ebe4d5eb1a5bd82740a822e...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/308456161?utm_source=github_status&utm_medium=notification)
Approved by: skystar-p
Pushing 8bcdef4755d7297c4ebe4d5eb1a5bd82740a822e to master...

**Pull request merged**